openedx_lti_tool_plugin/models.py

```diff
@@ -110,6 +111,9 @@ def family_name(self) -> str:
     @property
     def name(self) -> str:
         """str: Name."""
+        if name := self.pii.get('name', ''):
```

Comment:
```suggestion
       @property
def name(self) -> str:
    """str: Name."""
    # Return the name from pii if available
    name = self.pii.get('name', '')
    if name:
        return name

    # Construct the name based on available attributes
    parts = [self.given_name, self.middle_name, self.family_name]
    return ' '.join(part for part in parts if part) or self.given_name
```
@kuipumu What do you think?

Reply:
Nice!, Thanks.